app/index.html

```diff
@@ -368,38 +314,39 @@ <h2>Board of Directors</h2>
                </dl>
                <h2>Board Bios</h2>
                            
-               <p><b>Adam Kirsch</b> Bio in progress.</p>
+               <p><b>Adam Kirsch</b> Bio in progress. His term ends 2021.</p>
                
-               <p><b>Alan Petroff</b> Bio in progress</p>
+               <p><b>Alan Petroff</b> Bio in progress. His term ends 2021.</p>
                
                <p><b>Andrew P. Stifel</b> BA &#x2019;91, A&#x26;S, is a Senior Program Manager at Leidos.  His work in recent years has provided support to a variety of government clients primarily in the Defense and Intelligence sectors.  He has also provided analysis and support in the areas of Strategic Communication, Public Diplomacy, Political Instability, Intelligence Analysis, and Information Operations for a variety of clients. He majored in Government at Cornell.</p>
                
                <p>Andrew began fencing at DC Fencers Club in 1985 while at St. Albans School in Washington, DC, and went on to fence on the Varsity Foil team at Cornell with Jean-Jacques Gillet, Graeme Jennings and John Helmich, and served as Foil Captain his senior year.  Andrew went on to fence for Oxford University under Coach Tomek Walicki, earned his Full Blue representing the University, competed in a variety of international tournaments to include the World Student Games, and was ranked in the Top 50 fencers in the UK.  Upon returning to the U.S., Andrew competed for DCFC again, under Coach Janusz Smolenski, consistently ranked in the Top 50 foil fencers in the US, and went to US Nationals/Olympic Trials for 10 years running.  His highest rank in the US was 22nd.</p>
                
-               <p>He currently lives in Arlington, VA with his wife Amanda, daughter Emma (11) and son William (8), the latter of which has just recently taken up sabre.  Andrew coaches his daughter&#x2019;s soccer team, his son&#x2019;s baseball team, and he and his family enjoy a variety of outdoor pursuits, to include sailing, fishing, and hiking/climbing. Andrew is currently a board member. His first term ends in 2018.</p>
+               <p>He currently lives in Arlington, VA with his wife Amanda, daughter Emma (11) and son William (8), the latter of which has just recently taken up sabre.  Andrew coaches his daughter&#x2019;s soccer team, his son&#x2019;s baseball team, and he and his family enjoy a variety of outdoor pursuits, to include sailing, fishing, and hiking/climbing. Andrew is currently in his second term as Vice President of the board, his term ends in 2021.</p>

```

Comment:
Let's avoid run-ons. 
```suggestion
               <p>He currently lives in Arlington, VA with his wife Amanda, daughter Emma (11) and son William (8), the latter of which has just recently taken up sabre.  Andrew coaches his daughter&#x2019;s soccer team, his son&#x2019;s baseball team, and he and his family enjoy a variety of outdoor pursuits, to include sailing, fishing, and hiking/climbing. Andrew is currently in his second term as Vice President of the board. His term ends in 2021.</p>
```